### ISSUE TITLE
Error `not a package: "h" in h.Write <*ast.SelectorExpr>` when `h` is a variable and the code is otherwise valid

### ISSUE THREAD

**Issue by codemicro** [gopherdata/gophernotes#240]
When attempting to run the following snippet using Nteract and the latest version of GopherNotes, I get the following error:

```go
import "crypto/hmac"
import "crypto/sha1"

h := hmac.New(sha1.New, []byte("bananarama"))
_, _ = h.Write([]byte("some data"))
h.Sum(nil)
```
```
repl.go:5:8: not a package: "h" in h.Write <*ast.SelectorExpr>
```

However, if I try to compile and run the following using the `go run` command, I get no error, and it runs without a hitch:

```go
package main

import "crypto/hmac"
import "crypto/sha1"

func main() {
h := hmac.New(sha1.New, []byte("bananarama"))
_, _ = h.Write([]byte("some data"))
h.Sum(nil)
}
```

**Comment by cosmos72** [comment]
Confirmed, it's a bug. For some reason, the interpreter thinks `h` has no methods at all:
```Go
import "crypto/hmac"
import "crypto/sha1"

h := hmac.New(sha1.New, []byte("bananarama"))
h.Write
```
```
repl.go:1:1: type hash.Hash has no field or method "Write": h.Write
```
I will investigate this.

**Comment by codemicro** [comment]
Thanks for the quick response @cosmos72 :)

**Comment by cosmos72** [comment]
Fixed in version v0.7.4
